api/coralogix/v1alpha1/globalrouter_types.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	cxsdk "github.com/coralogix/coralogix-management-sdk/go"
 
+	coralogix_reconciler "github.com/coralogix/coralogix-operator/internal/controller/coralogix/coralogix-reconciler"
```

Comment:
```suggestion
	coralogixreconciler "github.com/coralogix/coralogix-operator/internal/controller/coralogix/coralogix-reconciler"
```

this follows the same pattern as other imports, e.g:
```
coralogixv1alpha1 "github.com/coralogix/coralogix-operator/api/coralogix/v1alpha1"
```